socorro/tests/external/boto/test_connection_context.py

```diff
@@ -2,56 +2,66 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+import os
+
 import pytest
 
-from socorro.external.boto.connection_context import S3Connection, KeyNotFound
-from socorro.tests.external.boto import get_config
+from socorro.external.boto.connection_context import KeyNotFound
+from socorro.libclass import build_instance_from_settings
+
+
+S3_SETTINGS = {
+    "class": "socorro.external.boto.connection_context.S3Connection",
+    "options": {
+        "region": os.environ["CRASHSTORAGE_S3_REGION"],
+        "access_key": os.environ["CRASHSTORAGE_S3_ACCESS_KEY"],
+        "secret_access_key": os.environ["CRASHSTORAGE_S3_SECRET_ACCESS_KEY"],
+        "endpoint_url": os.environ["AWS_ENDPOINT_URL"],
+    },
+}
 
 
 class TestS3ConnectionContext:
-    def test_save_file(self, boto_helper):
+    def test_save_file(self, s3_helper):
         """Test saving a file and make sure it's there."""
-        config = get_config(cls=S3Connection)
-        conn = S3Connection(config)
+        conn = build_instance_from_settings(S3_SETTINGS)
 
-        bucket = conn.config.bucket_name
+        bucket = os.environ["CRASHSTORAGE_S3_BUCKET"]
```

Comment:
I think `s3_helper` has a `get_crashstorage_bucket` method now. I'll update this.